src/foam/nanos/auth/resetPassword/EmailView.js

```diff
@@ -154,7 +154,7 @@ foam.CLASS({
         .start('p').addClass('link')
           .add('Sign in.')
           .on('click', function() {
-            self.stack.push({ class: 'foam.nanos.auth.SignInView' });
+            self.stack.push({ class: 'net.nanopay.sme.ui.SignInView' });
```

Comment:
This would be putting a reference to a NANOPAY-specific view into the foam2 repo, which is supposed to be a generic framework if I'm not mistaken. I don't think this makes sense to do.

Reply:
So another option would be create another forgot password view under nanopay.sme.ui

Reply:
Yeah. Another option would be to modify the foam EmailView to take a parameter for the sign in view it pushes onto the stack here that defaults to the foam one but can be set to a custom one if needed. I think that way avoids duplicating code.